frontend/.dockerignore

```
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.next
```

```suggestion
.next

```
元の私のが悪いんですがPOSIX仕様に合わせておいたほうが良いかなこれ

---

fix https://github.com/MISW/Portal/pull/386/commits/5f58c504f73cfdbc0c027992ec1da3beec5c82cf